.project-keeper.yml

```diff
@@ -5,7 +5,17 @@ sources:
       - jar_artifact
       - integration_tests
 build:
-  runnerOs: ubuntu-20.04
+  runnerOs: ubuntu-latest
   exasolDbVersions:
-    - "7.1.30"
-    - "8.32.0"
+    - "8.31.0" # Tests fail with 8.32.0, see https://github.com/exasol/virtual-schema-common-document/issues/193
```

Comment:
This issue affects only document based VS like S3. But this is JDBC based. Maybe upgrading to the latest virtual-schema-common-jdbc published recently fixes the issue?

Reply:
Seems you're right. No issues with 8.34 so far (here).
I've updated the CI version.